src/nvim/eval.c

```diff
@@ -514,6 +514,10 @@ void eval_init(void)
 
   for (size_t i = 0; i < ARRAY_SIZE(vimvars); i++) {
     p = &vimvars[i];
+    if (STRLEN(p->vv_name) > 16) {
+      EMSG("INTERNAL: name too long, increase size of dictitem16_T");
+      getout(1);
```

Comment:
Does it really matter?  This code is just here to flag an issue when a new item is added to the vimvars array.  The entire point is that you're supposed to notice that the variable name is too long and the array size should be changed.

Why not just change the if to `assert(STRLEN(p->vv_name) <= 16);`?  Then the getout and EMSG aren't needed.